src/akeneo/_twig/docker-compose.yml/application.yml.twig

```diff
@@ -0,0 +1,71 @@
+{% set blocks  = '_twig/docker-compose.yml/' %}
+{% set dockersync = false %}
+{% if @('host.os') == 'darwin' and @('docker-sync') == 'yes' %}
+{% set dockersync = true %}
+{% endif %}
+
+  console:
+{% if @('app.build') == 'dynamic' %}
+    build:
+      context: ./
+      dockerfile: .my127ws/docker/image/console/Dockerfile
+    entrypoint: [/entrypoint.dynamic.sh]
+    command: [sleep, infinity]
+    volumes:
+      - {{ (dockersync) ? @('workspace.name') ~ '-sync:/app:nocopy' : './:/app:delegated' }}
+      - ./.my127ws/application:/home/build/application
+      - ./.my127ws/docker/image/console/root/lib/task:/lib/task
+      - ./.my127ws:/.my127ws
+{% else %}
+    image: {{ @('docker.repository') ~ ':' ~ @('app.version') ~ '-console' }}
+{% endif %}
+    labels:
+      - traefik.enable=false
+    environment:
+{{ deep_merge_to_yaml([@('services.php-base.environment'), @('services.console.environment')], 2, 6) | raw }}
+{% include blocks ~ 'environment.yml.twig' %}
+    networks:
+      - private
+
+  nginx:
+{% if @('app.build') == 'dynamic' %}
+    build: .my127ws/docker/image/nginx
+    volumes:
+      - {{ (dockersync) ? @('workspace.name') ~ '-sync:/app:nocopy' : './:/app:delegated' }}
+{% else %}
+    image: {{ @('docker.repository') ~ ':' ~ @('app.version') ~ '-nginx' }}
+{% endif %}
+    labels:
+      - traefik.backend={{ @('workspace.name') }}
+      - traefik.frontend.rule=Host:{{ @('hostname') }}
+      - traefik.docker.network=my127ws
+      - traefik.port=80
+{% if @('services.nginx.environment') %}
+    environment:
+{{ to_nice_yaml(@('services.nginx.environment'), 2, 6) | raw }}
+{% endif %}
+    links:
+      - php-fpm:php-fpm
+    networks:
+      private:
+        aliases:
+          - {{ @('hostname') }}
+      shared: {}
+
+  php-fpm:
+{% if @('app.build') == 'dynamic' %}
+    build: .my127ws/docker/image/php-fpm
+    image: {{ @('workspace.name') ~ '-php-fpm:dev' }}
+    volumes:
+      - {{ (dockersync) ? @('workspace.name') ~ '-sync:/app:nocopy' : './:/app:delegated' }}
+      - ./.my127ws:/.my127ws
+{% else %}
+    image: {{ @('docker.repository') ~ ':' ~ @('app.version') ~ '-php-fpm' }}
+{% endif %}
+    labels:
+      - traefik.enable=false
+    networks:
+      - private
+    environment:
+{{ deep_merge_to_yaml([@('services.php-base.environment'), @('services.php-fpm.environment')], 2, 6) | raw }}
+{% include blocks ~ 'environment.yml.twig' %}
```

Comment:
feels a bit "spot the difference between this and _base's one". I can't see anything of significance

Reply:
it's the image name: 'image: {{ @('workspace.name') ~ '-php-fpm:dev' }}' so it can be used as the basis for the job-queue-consumer Dockerfile which only differs by one file (the supervisor config)

Reply:
I copied this approach from spryker and jenkins-runner, IIRC

Reply:
e.g. see console:dev https://github.com/inviqa/harness-spryker/blob/0.3.x/_twig/docker-compose.yml/application.yml.twig

Reply:
ok. I'd be curious if we can do something more standard later that could fit in the base. I have a few ideas on simplification of build process that might help